pyproject.toml

```diff
@@ -24,23 +23,25 @@ classifiers = [
     'Operating System :: Microsoft',
 ]
 homepage = 'https://github.com/sayanarijit/expandvars'
-authors = ['Arijit Basu <sayanarijit@gmail.com>']
+authors = [
+  {name = "Arijit Basu", email = "sayanarijit@gmail.com"}
+]
+maintainers = [
+  {name = "Arijit Basu", email = "sayanarijit@gmail.com"}
+]
 readme = 'README.md'
+license = {file = "LICENSE"}
+requires-python = ">=3"
 
-[tool.poetry.dependencies]
-python = '>=3.6'
+[project.urls]
+"Homepage" = "https://github.com/sayanarijit/expandvars"
 
-[tool.poetry.dev-dependencies]
-tox = '*'
-pytest = '*'
-pytest-cov = '*'
+[project.optional-dependencies]
+tests = ['tox', 'pytest', 'pytest-cov', 'toml']
```

Comment:
I think you also should remove `poetry` from `tox.ini` and add `extras = tests` there.

Reply:
Also `README.md` should be updated for the `poetry` -> `hatchling` change.

Reply:
I would strongly urge not depending on toml without a strong reason. Ecosystem is resolving into tomli and its API which is available in standard library since Python 3.11 as tomllib.

Reply:
Well, if my solution from https://github.com/sayanarijit/expandvars/pull/64#discussion_r1400918424 is used, then the test and therefore `toml` becomes unnecessary.